lib/bundle/dumper.rb

```diff
@@ -20,6 +20,7 @@ def self.dump_brewfile
       content << cask_before_formula
       content << BrewDumper.dump
       content << cask_after_formula
+      content << MacAppStoreDumper.dump
```

Comment:
You may want to use `mas` to install some apps. And you want to use `Brewfile` to manage dev env. That doesn't necessary mean you want to put them together. That's because your `Brewfile`(hence dev setup) is stable, but the list of **all** your mac apps is not.

Even more, I would publicly share my `Brewfile` in Github but not necessary mac apps list.